pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/DBClusterBasicOptions/DBClusterBasicOptions.tsx

```diff
@@ -21,6 +24,62 @@ export const DBClusterBasicOptions: FC<DBClusterBasicOptionsProps> = ({ kubernet
     change(AddDBClusterFields.databaseType, databaseType);
   }, []);
 
+  const kubernetesOptions = kubernetes.map((kubernetesCluster) => {
```

Comment:
useMemo here when kubernetes change might be useful to not run this on every render. I would even move this function to the utils file with a proper name (e.g. getKubernetesOptions ?) and split it (e.g. move the getOptionContent outside) so it gets easier to read.

Reply:
Good point, done